### ISSUE TITLE
On applying style created in dt 3.0 receive "module basicadj version mismatch 2!=1"

### ISSUE THREAD

**Issue by cigno5** [darktable-org/darktable#4161]
I created early December 2019 a preset for a certain kind of pictures. The preset included basic adjustment module. Today with the most recent commit, when I apply the style, I get **"module basicadj version mismatch 2!=1"**. DT continues to work but the basic adjustments module is disabled and settings haven't been applied.

It happens the same when I re-open an old edit (as bug #4059 ) or I clone the same old edit. The edits were done using basic adjustments module 

 - Darktable commit: 304f71e37
 - OS: Linux Ubuntu 18.04

Here it is the style `style1.dtstyle`:

```xml
<?xml version="1.0" encoding="UTF-8"?>
<darktable_style version="1.0">
    <info>
        <name>Style 1</name>
        <description>Style 1</description>
    </info>
    <style>
        <plugin>
            <num>9</num>
            <module>5</module>
            <operation>exposure</operation>
            <op_params>0000000000000000202f1d3f00004842000080c0</op_params>
            <enabled>1</enabled>
            <blendop_params>gz11eJxjYGBgkGAAgRNODGiAEV0AJ2iwh+CRyscOAAdeGQQ=</blendop_params>
            <blendop_version>9</blendop_version>
            <multi_priority>0</multi_priority>
            <multi_name></multi_name>
            <iop_order>21,000000</iop_order>
        </plugin>
        <plugin>
            <num>10</num>
            <module>3</module>
            <operation>colorbalance</operation>
            <op_params>010000000000803f0000803f0000803f0000803f0000803fb96b813f8e287d3f8e287d3f0000803f0000803f0000803f0000803f0000803f0000803f0000904137f88a3f</op_params>
            <enabled>1</enabled>
            <blendop_params>gz11eJxjYGBgkGAAgRNODGiAEV0AJ2iwh+CRyscOAAdeGQQ=</blendop_params>
            <blendop_version>9</blendop_version>
            <multi_priority>0</multi_priority>
            <multi_name></multi_name>
            <iop_order>40,000000</iop_order>
        </plugin>
        <plugin>
            <num>11</num>
            <module>1</module>
            <operation>basicadj</operation>
            <op_params>00000000d0cccc3e00000000000000000000000001000000295c93417c142ebe0000000000000000</op_params>
            <enabled>1</enabled>
            <blendop_params>gz11eJxjYGBgkGAAgRNODGiAEV0AJ2iwh+CRyscOAAdeGQQ=</blendop_params>
            <blendop_version>9</blendop_version>
            <multi_priority>0</multi_priority>
            <multi_name></multi_name>
            <iop_order>39,000000</iop_order>
        </plugin>
        <plugin>
            <num>12</num>
            <module>4</module>
            <operation>colorzones</operation>
            <op_params>gz02eJxjYoADex4Bbjupbyz2Rw8a2i3j0bAPSIi2u7vV0N7frMbOJobRnoGhwR6kjgENtACxQKZ6VmUougwDw4Hw2mxk8f9QAOMzQjFMP0gORG94ElYVVg+UYJr3FOa+PQ9abC/n8tirGWjZXbxqZr+/pd/uGvt/O5i7Qq/PuntArgJsHyOaO0xg7rmRA5bngPI3QPkMf9JjQPoVgExmIF5w6yrYPQLs857+A7pjxaGDYP4CKB2RqgjWFwCkYe6DucMAZA8TRB8DjIaanwdkRgPxCnlIOLBCNRvEQ1ysAOXD7LuwYTbYnszU529A4REAtTcnCsIHqV+ok5h3AWrPgtMvwOIODXrpsHBnA2KQPUxoGAYA5vaBeQ==</op_params>
            <enabled>1</enabled>
            <blendop_params>gz11eJxjYGBgkGAAgRNODGiAEV0AJ2iwh+CRyscOAAdeGQQ=</blendop_params>
            <blendop_version>9</blendop_version>
            <multi_priority>0</multi_priority>
            <multi_name></multi_name>
            <iop_order>58,000000</iop_order>
        </plugin>
        <plugin>
            <num>13</num>
            <module>1</module>
            <operation>atrous</operation>
            <op_params>gz05eJxjZoCAs2d87M6eOWM3a6akPZBtz8DQYE8HcTg+4s1ozztf3/6NNgOKOJGYJAAAE/g1eA==</op_params>
            <enabled>1</enabled>
            <blendop_params>gz11eJxjZGBgYGIAgRNODGiAEV0AJ2iwh+CRyscOAOsbGO8=</blendop_params>
            <blendop_version>9</blendop_version>
            <multi_priority>0</multi_priority>
            <multi_name></multi_name>
            <iop_order>32,000000</iop_order>
        </plugin>
    </style>
</darktable_style>
```

**Comment by achienbsi** [comment]
This happened to me with styles I created using DT 3.0 Windows. After upgrading to 3.0.1 all those images give me this error. Any way to fix this without re-editing all the photos?

**Comment by aurelienpierre** [comment]
Fixed by 48b7e80
